presto-spark-base/src/test/java/com/facebook/presto/spark/execution/common/TestProperties.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.common;
+
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+public class TestProperties
+{
+    @Test
+    public void testFilePropertiesPopulator()
+    {
+        Path directory = null;
+        try {
+            directory = Files.createTempDirectory("presto");
+        }
+        catch (IOException e) {
+            e.printStackTrace();
+            fail();
+        }
+        WorkerConfigProperties.WorkerConfigPropertiesBuilder configPropertiesBuilder = new WorkerConfigProperties.WorkerConfigPropertiesBuilder();
```

Comment:
We probably can have a common method taking the `Properties` class (the base class for all properties) and do the verification, we just need to call this method three times for different properties implementation.